mantle/platform/api/aliyun/api.go

```diff
@@ -385,3 +387,41 @@ func (a *API) ListRegions() ([]string, error) {
 	sort.Strings(ret)
 	return ret, nil
 }
+
+// ChangeVisibility modifies an image uploaded to Aliyun as either public or
+// private.
+// NOTE: only us-east-1 and us-west-1 support making images public unless
+// your account has been allowlisted by Aliyun to operate on all regions
+func (a *API) ChangeVisibility(region string, id string, visibility bool) error {
```

Comment:
optional:

```suggestion
func (a *API) ChangeVisibility(region string, id string, public bool) error {
```

Since this is a boolean we can just name it `public` and then let `public = False` mean private.